tests/test_parseOCLC.py

```diff
@@ -47,36 +42,6 @@ def test_holdings_bad_uri(self):
         extractHoldingsLinks([mock_holding], mock_instance)
         mock_add.assert_not_called()
 
```

Comment:
Does there need to be another test for adding a book with a blacklisted uri?

Reply:
A good idea